numpy/random/_generator.pyx

```diff
@@ -396,6 +396,26 @@ cdef class Generator:
         out : ndarray or scalar
             Drawn samples from the parameterized beta distribution.
 
+        Examples
+        --------
+        Scalar values for `a` and `b`, size = 3, results
+        in list with 3 samples from beta distribution
+
+        >>> rng = np.random.default_rng()
+        >>> rng.beta(0.5, 0.5, 3)
+        array([0.07092512, 0.59868941, 0.21869789])  # random
+        
+        Note the row sublists and column sublists for `a`
+        and `b` respectively below
+        Vector values for `a` and `b`, results in 3 * 2 ndarray
+        with singular samples for each pairwise combination
+        of `a` and `b`
+
+        >>> rng = np.random.default_rng()
+        >>> rng.beta([[0.5], [0.25], [0.75]], [[0.5, 0.25]])
+        array([[6.26304645e-01, 3.57283854e-02],
+                [1.40768012e-02, 1.72984613e-05],
+                [4.63162592e-01, 3.38723747e-01]])  # random
```

Comment:
```suggestion
```

I would delete this paragraph and the following example. The broadcasting behavior that's being demonstrated is general to all of the methods, and we don't really need to demonstrate it just for `beta`. And "sublist" is a novel term that we don't use anywhere else; not entirely sure what that's referring to.

What's left doesn't really add much. Generally, in these examples, we try to use the example to point out something interesting about the relationship between the parameters and the generated values (e.g. that we can reproduce the mean and variance of a normal distribution). A kind of minimum would be to [plot a histogram](https://numpy.org/doc/stable/reference/random/generated/numpy.random.Generator.triangular.html).